src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/InvokeRestMethodCommand.Common.cs

```diff
@@ -85,7 +85,7 @@ public int MaximumFollowRelLink
 
```

Comment:
@iSazonov, your last commit had 164 failures in `PowerShell-CI-windows`
(These are 5 of the failures)

Web cmdlets tests using the cmdlet's aliases.Execute Invoke-RestMethod
```powershell
Specified method is not supported.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\WebCmdlets.Tests.ps1: line 3476
3476:         $result = Invoke-RestMethod $uri

```
Invoke-RestMethod tests.Invoke-RestMethod returns Correct User-Agent on Windows
```powershell
Expected regular expression '.*\(Windows NT \d+\.\d*;.*\) PowerShell\/\d+\.\d+\.\d+.*' to case sensitively match $null, but it did not match.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\WebCmdlets.Tests.ps1: line 1995
1995:         $result.Output.headers.'User-Agent' | Should -MatchExactly '.*\(Windows NT \d+\.\d*;.*\) PowerShell\/\d+\.\d+\.\d+.*'
```
Invoke-RestMethod tests.Invoke-RestMethod returns headers dictionary
```powershell
Expected '127.0.0.1:8083', but got $null.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\WebCmdlets.Tests.ps1: line 2005
2005:         $result.Output.headers.Host | Should -Be $Uri.Authority
```
Invoke-RestMethod tests.Validate Invoke-RestMethod -DisableKeepAlive
```powershell
Expected '127.0.0.1:8083', but got $null.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\WebCmdlets.Tests.ps1: line 2016
2016:         $result.Output.headers.Host | Should -Be $uri.Authority
```
Invoke-RestMethod tests.Validate Invoke-RestMethod -MaximumRedirection
```powershell
Expected regular expression '127.0.0.1:8083' to match $null, but it did not match.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\WebCmdlets.Tests.ps1: line 2027
2027:         $result.Output.headers.Host | Should -Match $uri.Authority
```